server/models/video/video-file.ts

```diff
@@ -29,6 +31,33 @@ import { MVideoFile, MVideoFileStreamingPlaylistVideo, MVideoFileVideo } from '.
 import { MStreamingPlaylistVideo, MVideo } from '@server/typings/models'
 import * as memoizee from 'memoizee'
 
+export enum ScopeNames {
+  WITH_VIDEO = 'WITH_VIDEO',
+  WITH_METADATA = 'WITH_METADATA'
+}
+
+const METADATA_FIELDS = [ 'metadata', 'metadataUrl' ]
+
+@DefaultScope(() => ({
+  attributes: {
+    exclude: METADATA_FIELDS
```

Comment:
I think we need `metadataUrl`, or we won't be able to fetch metadata no?

I think your tests work because most of the time we use `VideoFileModel.unscoped()` in joins. I think you can remove the `unscoped()` call

Reply:
Do you mean to remove the `unscoped()` call line 50?

Reply:
No, I mean removing `VideoFileModel.unscoped()` in `models/video/video.ts` and `models/redundancy/video-redundancy.ts`